dom_wasm.go

```diff
@@ -1088,14 +1088,16 @@ func (w *window) Stop() {
 
 // TODO reuse util.EventTarget
 
-func (w *window) AddEventListener(typ string, useCapture bool, listener func(Event)) func(o js.Value) {
-	wrapper := func(o js.Value) { listener(wrapEvent(o)) }
-	w.Call("addEventListener", typ, wrapper, useCapture)
-	return wrapper
+func (w *window) AddEventListener(typ string, useCapture bool, listener func(Event)) js.Callback {
+	wrapper := func(o []js.Value) { listener(wrapEvent(o[0])) }
+	cb := js.NewCallback(wrapper)
+	w.Call("addEventListener", typ, cb, useCapture)
+	return cb
 }
 
-func (w *window) RemoveEventListener(typ string, useCapture bool, listener func(js.Value)) {
+func (w *window) RemoveEventListener(typ string, useCapture bool, listener js.Callback) {
```

Comment:
there's a comment in the syscall/js code that says removing can only work if you remove the exact object you add, so this should be RemoveEventListener(cb js.Callback)

Reply:
then fix the line below to use the remove func from syscall/js

Reply:
Please link the comment around callback removal.

Can you provide an example of the change you are expecting?